### ISSUE TITLE
db.list command doesn't work on odoo 12.0

### ISSUE THREAD

**Issue by zeroincombenze** [OCA/odoorpc#34]
Hi,
I am testing odoorpc and Odoo 12.0 but i meet error 404.
Set to reproduce:
```
>>> import odoorpc
>>> odoo = odoorpc.ODOO('localhost', 'jsonrpc', 8069)
>>> print odoo.version
12.0
>>> odoo.db.list()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/site-packages/odoorpc/db.py", line 273, in list
    'args': []})
  File "/usr/lib/python2.7/site-packages/odoorpc/odoo.py", line 278, in json
    data = self._connector.proxy_json(url, params)
  File "/usr/lib/python2.7/site-packages/odoorpc/rpc/jsonrpclib.py", line 91, in __call__
    response = self._opener.open(request, timeout=self._timeout)
  File "/usr/lib64/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/usr/lib64/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python2.7/urllib2.py", line 475, in error
    return self._call_chain(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 558, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 404: NOT FOUND
```
Same code works with 11.0 and 10.0

**Comment by pedrobaeza** [comment]
Haven't you tried any other command, like login or perform other action?

I have change title to adjust to current issue report.

**Comment by zeroincombenze** [comment]
db.create() works. I am testing for the other commands.

**Comment by zeroincombenze** [comment]
Other commands seems to work.
I tested for search, browse, write and some others.
PS currently odoorpc tested version is 0.6.2. Previosly I tested 0.6.1

**Comment by zeroincombenze** [comment]
Hi guys,
I completed test of my code https://github.com/zeroincombenze/tools/tree/master/clodoo.
This library use odoorpc to work so I can give a complete set of test results:
- db.create => Ok
- db.list => FAILS
- db.drop => FAILS
- create() => Ok
- write() => Ok
- search() => Ok
- browse() => Ok
- execute() => Ok

For furthermore info contact me.
HTH

**Comment by sebalix** [comment]
@zeroincombenze I still didn't release the latest code which adds the support for Odoo 12 (`db.list()` should work if I trust this test on Odoo 12: https://travis-ci.org/OCA/odoorpc/jobs/453210935#L780).
Can you test the branch `master` to see if it fixes your problem? If not I will investigate to fix that before the new release.

**Comment by zeroincombenze** [comment]
Dear @sebalix,
how I can test master branch?
I tested 0.6.2  is not the latest version?

**Comment by sebalix** [comment]
To install it:
```bash
$ git clone https://github.com/OCA/odoorpc.git
$ pip install -e odoorpc/
```

**Comment by zeroincombenze** [comment]
Hi @sebalix,
I execute above statements to install odoorpc but I always meet error 404 for all function.

It is strange. I am not sure if I tested right version, because command <pip show> tell me "0.6.2", exactly the same version after I issued <pip installl odoorpc -U> command.

**Comment by zeroincombenze** [comment]
I cannot test new module in my unit test.
db.list() work but I could not test dropdb()
Sorry! :-(

**Comment by Odoo-Italia-Associazione** [comment]
Hi,
I discovered another minor trouble: browse() ignore context.
i.e.
if user set with Italian language it_IT, do execute follow statement

     odoo.browse('res.groups', id, context={'lang': 'en_US'}).name

returned name is Italian text, no English text.

**Comment by sebalix** [comment]
Hi @Odoo-Italia-Associazione can you report it in another issue please? Easier to track.

**Comment by sebalix** [comment]
@zeroincombenze is your issue resolved?

**Comment by zeroincombenze** [comment]
Yes, I solved with odoorc 7.0
Thank you
